frontend/src/app/app.component.ts

```diff
@@ -4,7 +4,7 @@ import { SpinnerService } from '@core/services/spinner.service';
 
 @Component({
     selector: 'app-root',
-    template: '<router-outlet></router-outlet>'
+    template: '<toast-notification></toast-notification><router-outlet></router-outlet>'
```

Comment:
You can put "p-toast" here instead of your wrapper.

Reply:
I tried and its doesn't work. PrimeNG have MessageService that work with a component <p-toast>. But we need wrapper under this MessageService. We can work always with MessageService, but we will always import modules and apply config. So i decided to create my own component that injects into "main" and work with notifs from a service. I have used this doc https://primefaces.org/primeng/showcase/#/toast

Reply:
And also if we use our wrapper, we can make a specific methods and use them anywhere.